apps/examples/react-ssr-example/src/shared/components/atoms/Link/Link.tsx

```diff
@@ -1,34 +1,24 @@
 import React from "react";
 
 interface LinkProps {
-    href: string | string[];
+    href?: string | string[];
```

Comment:
When wouldn't a link require an href?

Reply:
Mostly never but the reason for this was to simplify the conditions in the template. I kinda debate back and forth on stuff like this. It allows the component to take in non-value and just not render anything.